tweb/src/main/resources/templates/index.html

```diff
@@ -83,7 +84,37 @@ <h1 class="home-title">
                 </div>
             </div>
         </div>
+        <div class="container my-5 px-5">
+            <div class="row">
+                <div class="col-12 col-md-3 col-lg-3 offset-md-2">
+                    <img style="width: 87%;"
+                         src="https://user-images.githubusercontent.com/6882032/41536582-52e43a30-7306-11e8-92df-2fe6502ad760.png"/>
+                </div>
+                <div class="col-12 col-md-7 col-lg-7 help-text">
+                    <h2 class="help-text__header">
+                        Have a question or need some help?
+                        <br/>
```

Comment:
No <br/> for spacing, only for breaking text when really needed.

Reply:
done